portfolio/src/main/webapp/contact/css/main.css

```diff
@@ -0,0 +1,380 @@
+
+
+
+
+/*//////////////////////////////////////////////////////////////////
+[ FONT ]*/
+
+@font-face {
+  font-family: Ubuntu-Bold;
+  src: url('../fonts/ubuntu/Ubuntu-Bold.ttf'); 
+}
+
+
+

```

Comment:
same here, there are lots of extra new lines. Lets keep it to 1 new line in between font/class declarations